winit-core/src/event.rs

```diff
@@ -820,47 +820,47 @@ impl Modifiers {
         Self { state, pressed_mods }
     }
 
-    /// The state of the modifiers.
+    /// The logical state of the modifiers.
     pub fn state(&self) -> ModifiersState {
         self.state
     }
 
-    /// The state of the left shift key.
+    /// The logical state of the left shift key.
     pub fn lshift_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::LSHIFT)
     }
 
-    /// The state of the right shift key.
+    /// The logical state of the right shift key.
     pub fn rshift_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::RSHIFT)
     }
 
-    /// The state of the left alt key.
+    /// The logical state of the left alt key.
     pub fn lalt_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::LALT)
     }
 
-    /// The state of the right alt key.
+    /// The logical state of the right alt key.
     pub fn ralt_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::RALT)
     }
 
-    /// The state of the left control key.
+    /// The logical state of the left control key.
     pub fn lcontrol_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::LCONTROL)
     }
 
-    /// The state of the right control key.
+    /// The logical state of the right control key.
     pub fn rcontrol_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::RCONTROL)
     }
 
-    /// The state of the left super key.
+    /// The logical state of the left super key.
     pub fn lsuper_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::LMETA)
     }
 
-    /// The state of the right super key.
+    /// The logical state of the right super key.
     pub fn rsuper_state(&self) -> ModifiersKeyState {
         self.mod_state(ModifiersKeys::RMETA)
     }
```

Comment:
Nit: `ModifiersKeyState` already documents that it is logical; it feels a bit redundant to do so here too?

Reply:
But then you'd have to lose focus with an extra jump to another doc page?

Reply:
Maybe document it on `Modifiers` then? But really, I'm fine with it either way.